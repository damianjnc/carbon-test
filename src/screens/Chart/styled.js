// @flow

import styled from 'styled-components'

export const Wrapper = styled.div`
  width: 80%;
  display: flex;
  flex-direction: column;
  align-items: center;
`