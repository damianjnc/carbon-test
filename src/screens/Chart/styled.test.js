// @flow

import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import { Wrapper } from './styled'

describe('screens > Chart', () => {
  describe('Wrapper', () => {
    it('renders', () => {
      const wrapper = renderer.create(<Wrapper />).toJSON()

      expect(wrapper).toMatchSnapshot()
    })
  })
})
