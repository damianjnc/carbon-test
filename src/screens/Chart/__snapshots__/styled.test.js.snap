// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screens > Chart Wrapper renders 1`] = `
.c0 {
  width: 80%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
/>
`;
